runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -90,4 +90,21 @@ default boolean syncPanel()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "ignoredEvents",
+		name = "Ignored groups",
+		description = "Configures which loot groups should be excluded from the panel UI"
+	)
+	default String getIgnoredEvents()
```

Comment:
shouldn't this be hidden?

Reply:
I didn't hide this as the config option for hiding items is also not hidden.

I'm pretty sure having it hidden could cause issues with the limiting that would prevent someone from re-including an event.